.env

```diff
@@ -1,2 +1,3 @@
 COMPOSE_PROJECT_NAME=dojot
 FLOWBROKER_NETWORK=flowbroker
+SERV_PORT=80
```

Comment:
Is this port intended to be used for other services as well? If not, I'd give it a more specific name, such as `DATA_BROKER_SERVICE_PORT`.